### PR TITLE
refactor(module): update module

### PR DESCRIPTION
**Pull request by ricardogobbosouza**
* Update module following `module-template`
* Coverage works again :smile:

Resolves #17 
Resolves #15 
Resolves #23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,26 +17,29 @@ Redirecting URLs is an often discussed topic, especially when it comes to
...
+    // Simple usage
+    ['@nuxtjs/redirect-module', {
+      // Redirect option here
+    }]
  ]
 }
 ```
 
-### Dedicated option array
+using top level options
```
Comment (manniL) [inline README.md]: Improve subheading please

**Comment by ricardogobbosouza** [comment]
@manniL Can merged?

File: README.md [hunk 2]
```diff
@@ -16,26 +17,29 @@ Redirecting URLs is an often discussed topic, especially when it comes to
...
-### Inline options
+1. Add `@nuxtjs/redirect-module` dependency with `yarn` or `npm` into your project
+2. Add `@nuxtjs/redirect-module` to `modules` section of `nuxt.config.js`:
+3. Configure it:
+
 ```js
 {
   modules: [
-    ['@nuxtjs/redirect-module', [ /* Redirect option here */]],
+    // Simple usage
```
Comment (manniL) [inline README.md]: Why do we have that "Simple usage" comment everywhere? :thinking:

File: lib/module.js [hunk 3]
```diff
@@ -14,3 +15,6 @@ export default async function nuxtRedirect(moduleOptions) {
 
   this.addServerMiddleware(middleware)
 }
+
+module.exports = redirectModule
```
Comment (manniL) [inline lib/module.js]: if it's ESM syntax, keep it ESM syntax.

File: package.json [hunk 4]
```diff
@@ -1,73 +1,53 @@
 {
   "name": "@nuxtjs/redirect-module",
   "version": "0.2.0",
-  "description": "",
+  "description": "Redirect Module makes the redirects will become easier.",
```
Comment (manniL) [inline package.json]:
```suggestion
  "description": "Nuxt module to dynamically redirect initial requests",
```

Maybe?

Comment (ricardogobbosouza) [reply package.json]: Changed, my english is not very good :worried:

File: README.md (hunk 2 above)
Comment (ricardogobbosouza) [reply README.md]: Removed, no need :+1:

File: lib/module.js (hunk 3 above)
Comment (ricardogobbosouza) [reply lib/module.js]: I want to do in the next PR the change to ESM

File: package.json (hunk 4 above)
Comment (manniL) [reply package.json]: @ricardogobbosouza Practice is key! No worries :)

File: lib/module.js (hunk 3 above)
Comment (manniL) [reply lib/module.js]: ` export default async function nuxtRedirect(moduleOptions) {`

It *was* already ESM

Comment (ricardogobbosouza) [reply lib/module.js]: If it is not ESM we open coverage for nuxt 1x

Comment (ricardogobbosouza) [reply lib/module.js]: In my opinion we should release a version for Nuxt 1x and later a major release only covering Nuxt 2x

Comment (manniL) [reply lib/module.js]: I agree as mentioned before.

Forgot that this plugin didn't include a notice in the README, so yeah, let's do it the way you suggested.

**Pull request merged**